lib/menu.rb

```diff
@@ -0,0 +1,21 @@
+class Menu
+
+  attr_reader :dishes, :seleted_dishes
+
+  def initialize
+    @dishes = { "Pizza" => "7.99", "Chips" => "1.50", "Burger" => "4.99" }
+    @basket = []
+  end
+
+  def print_dishes
+    @dishes.each do |food, price|
+      puts "#{food}: £#{price}"
+    end
+  end
+
+  def order(*choices)
+    choices.select do |choices|  
```

Comment:
What do you think about replacing these two lines with the one liner `choices.select {|x|@dishes.keys.include?(x)}` ?